RustBridge/RustBridgeImplementation.cpp

```diff
@@ -294,8 +294,10 @@ namespace WPEFramework {
 					
 					searchPath.push_back(framework->DataPath());
 					searchPath.push_back(framework->PersistentPath());
-					searchPath.push_back(framework->SystemPath());
-					searchPath.push_back(framework->PluginPath());
+					//searchPath.push_back(framework->SystemPath());
+					//searchPath.push_back(framework->PluginPath());
+					searchPath.push_back(framework->VolatilePath());
+					searchPath.push_back("/usr/lib/wpeframework/plugins/");
```

Comment:
This is not the recommended way to hardcode paths in the code. It will lead to specific code deployments and hard to reuse code. I think the path indicated here is the path that is under compilations from recipes exactly the SystemPath, so why would you comment the system path and hard code it here ?

Reply:
removed the hard coded path